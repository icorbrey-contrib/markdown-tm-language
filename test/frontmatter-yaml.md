---
yaml: 1
---

# hi
