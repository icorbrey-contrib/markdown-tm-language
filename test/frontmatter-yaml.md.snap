>---
#^^^ source.md punctuation.definition.string.start.yaml
>yaml: 1
#^^^^^^^^ source.md meta.embedded.yaml
>---
#^^^ source.md punctuation.definition.string.end.yaml
>
># hi
#^ source.md markup.heading.atx.1.md punctuation.definition.heading.md
# ^ source.md markup.heading.atx.1.md
#  ^^ source.md markup.heading.atx.1.md entity.name.section.md
>